auslib/db.py

```diff
@@ -2529,6 +2527,7 @@ def bleet(table, type_, changed_by, query, transaction):
                 body.append(UTF8PrettyPrinter().pformat(row))
         elif type_ == "INSERT":
             body.append("Row to be inserted:")
+            query.parameters['rule_id'] = inserted_item_id
```

Comment:
Yeah, this is a bit problematic - we can't reference table-specific columns here (otherwise we end up putting things like "rule_id" in e-mails about tables that don't have one. The primary key column names either need to be passed into make_change_notifier or whenever onInsert is called. I think former would be a bit cleaner.

Reply:
Noted..working on this